site/docs/red-team/owasp-llm-top-10.md

```diff
@@ -153,70 +220,113 @@ redteam:
     - owasp:llm:06
 ```
 
-## 7. Insecure Plugin Design (LLM07)
+You can learn more about red teaming agents in [Promptfoo's guide](/docs/red-team/agents/).
 
-Promptfoo provides several plugins to test for vulnerabilities related to insecure plugin design:
+## 7. System Prompt Leakage (LLM07)
 
-- BOFA (Broken Object-Level Authorization) Testing
-- BFLA (Broken Function-Level Authorization) Testing
-- RBAC (Role-Based Access Control) Testing
-- SQL injection
-- Shell injection
-- Debug access
+System prompts are instructions provided to an LLM that guide the behavior of the model. They are designed to instruct the LLM based on application requirements. In some cases, system prompts may contain sensitive information that is not intended to be disclosed to the user or even contain secrets.
 
-As a rule of thumb, think about the [OWASP Top 10 for API](https://owasp.org/www-project-api-security/) when thinking about how to test for insecure plugin design.
-
-Example configuration:
+Promptfoo provides a plugin to test for prompt extraction:
 
 ```yaml
 redteam:
   plugins:
-    - rbac # Role-Based Access Control
-    - bofa # Broken Object-Level Authorization (aka IDOR-type vulnerabilities)
-    - bfla # Broken Function-Level Authorization
-    - sql-injection
-    - shell-injection
-    - debug-access
+    - id: 'prompt-extraction'
+      config:
+        systemPrompt: 'You are a homework assistant. You are given a homework problem and your job is to solve it. You will be given the homework problem and the answer. You will then return the solution to the problem.'
 ```
 
-Or, using the OWASP shorthand:
+:::note
+The `systemPrompt` config is required. It is the system prompt you provided to the model to instruct it how to act.
+:::
+
+## 8. Vector and Embedding Weaknesses (LLM08)
+
+OWASP defines vector and embedding vulnerabilities as weaknesses in how vectors and embeddings are generated, stored, or retrieved within the context of Retrieval Augmented Generation (RAG). Promptfoo supports RAG testing through multiple configurations:
+
+### Testing for Access Control
+
+Evaluate your RAG application against access control misconfigurations in your RAG architecture:
 
 ```yaml
 redteam:
   plugins:
-    - owasp:llm:07
+    - rbac # Role-Based Access Control
+    - bola # Broken Object-Level Authorization (aka IDOR-type vulnerabilities)
+    - bfla # Broken Function-Level Authorization
 ```
 
-## 8. Excessive Agency (LLM08)
+### Context Injection
```

Comment:
We should show the `indirect-prompt-injection` plugin here - that is the same as context injection